Form/Type/DocumentType.php

```diff
@@ -66,16 +66,22 @@ public function setDefaultOptions(OptionsResolverInterface $resolver)
             return $registry->getManager($manager);
         };
 
-        $resolver->setNormalizers(array(
-            'em' => $normalizer,
-        ));
+        $resolver->setNormalizer('em', $normalizer);
     }
 
     /**
-     * @see Symfony\Component\Form\FormTypeInterface::getName()
+     * {@inheritdoc}
      */
     public function getName()
     {
+        return $this->getBlockPrefix();
```

Comment:
Reading through the [UPGRADE document for 2.8](https://github.com/symfony/symfony/blob/2.8/UPGRADE-2.8.md) we have to return what is now the block prefix here for BC reasons.